lib/ansible/modules/monitoring/pagerduty_alert.py

```diff
@@ -26,13 +26,18 @@
 options:
     name:
         description:
-            - PagerDuty unique subdomain.
+            - PagerDuty unique subdomain. Obsolete. It is not used with PagerDuty REST v2 API.
+    service_id:
+        description:
+            - ID of PagerDuty service when incidents will be triggered, acknowledged or resolved.
         required: true
-    service_key:
```

Comment:
Is service key no longer a parameter that's accepted? To help with backwards compatibility, maybe we could add this as an alias of `integration_key`